app/views/rejected_notification_errors.scala.html

```diff
@@ -36,51 +36,79 @@
     notificationErrorHelper: NotificationErrorHelper
 )
 
-@(declaration: ExportsDeclaration, maybeMrn: Option[String], errors: Seq[NotificationError])(implicit request: Request[_], messages: Messages)
+@(declaration: ExportsDeclaration, maybeMrn: Option[String], maybeSubmissionId: Option[String], errors: Seq[NotificationError])(implicit request: Request[_], messages: Messages)
```

Comment:
would it make more sense to call this param `amendmentSubmissionId` just to make obvious that it's not any old submissionId getting passed (or not) here?